hybrid/src/main/java/jetbrains/jetpad/hybrid/TokenCompleter.java

```diff
@@ -126,10 +126,15 @@ public Runnable complete(int selectionIndex, Token... tokens) {
         final boolean wasCompletionActive = controller != null && controller.isActive();
 
         TokenListEditor<?> tokenListEditor = getTokenListEditor();
-        tokenListEditor.tokens.remove(index);
-        int i = index;
-        for (Token t : tokens) {
-          tokenListEditor.tokens.add(i++, t);
+
+        if (tokens.length == 0) {
+          tokenListEditor.tokens.remove(index);
+        } else {
+          int targetIndex = index, sourceIndex = 0;
```

Comment:
This is not according to the Java code style. There should be one declarations per line.
See http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141270.html#18761